### DISCUSSION TITLE
move the minimize maximize and close buttons to a custom titlebar

### DISCUSSION THREAD

**Discussion by zeridev** [Nectres/animos#8] [Ideas]
Moving the minimize maximize and close buttons to a custom titlebar for a cleaner look.
these can just be icon buttons with events on the main process.
to still be able to drag the application add webkit-app-region: drag to the custom titebar.

**Comment by keerthivasansa** [comment]
That's a neat idea. The app already has navigation bar so adding the minimize and maximize to the navigation bar should make the app look cleaner.

**Reply by zeridev** [reply]
i cloned the repository and i am working on this myself, but this is my first time contributing to a project so i dont know how to submit it for review.
i really like the project and would also be interested in continuing to work on it.

**Reply by keerthivasansa** [reply]
It's nice to see people are interested to work on the project!

If you want to contribute, fork the repo instead of cloning it, create a new branch, make the necessary changes, test it, open a pull request to my master branch (Nectres/master)

Then I can review the code and merge with master branch
